modules/oadp-configuring-custom-velero-plugins.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: PROCEDIRE
+[id="oadp-configuring-custom-velero-plugins_{context}"]
+= Configuring custom OADP Velero plug-ins
+
+You can install a custom Velero plug-in by specifying the plug-in `image` and `name` when you configure the `oadp_v1alpha1_dpa.yaml` file during deployment.
+
+.Procedure
+
+* Specify the desired plug-ins in the `oadp_v1alpha1_dpa.yaml` file during deployment. The file that follows installs the default `azure` and `gcp` plug-ins and a custom plug-in the name `custom-plugin-example` and the image `quay.io/example-repo/custom-velero-plugin`:
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+ name: dpa-sample
+spec:
+ configuration:
+   velero:
+     defaultPlugins:
+     - azure
```

Comment:
I would add openshift plugin in the examples by default. Here and also in other modules.